metricbeat/module/docker/container/data.go

```diff
@@ -31,7 +32,26 @@ func eventMapping(cont *dc.APIContainers) common.MapStr {
 		"status": cont.Status,
 	}
 
+// Check id container have health metrics configured
+if strings.Contains(cont.Status, "(") && strings.Contains(cont.Status, ")") {
+	container, _ := m.dockerClient.InspectContainer(cont.ID)
```

Comment:
We could just introduce a configuration option as mentioned above with `container.health: false` by default. Alternatively we can make it an additional metricset. I like the idea of a separate metricset and we can still merge it into container later if needed. 👍